libcontainer/init_linux.go

```diff
@@ -140,6 +142,30 @@ func finalizeNamespace(config *initConfig) error {
 	return nil
 }
 
+// lateCgroups applies any cgroup limits which must be done as late as possible.
+// Specifically, this refers to pids.max which runs into issues when Go likes to
+// start spawning new OS threads seemingly at random.
+func lateCgroups(config *initConfig) error {
+	if path, ok := config.CgroupPaths["pids"]; ok {
+		// XXX: We have to bootstrap this here because the manager hasn't joined
```

Comment:
I don't like the `MkdirAll` here. It would be nice if the manager actually `Apply`'d the cgroups (thus generating all of the directories and setting the limits) before this point. Why isn't the manager running at this point, by the way? The child has the config, so the manager must've run `manager.Apply(p.pid())` right?